content/en/watchdog/insights.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Watchdog Insights
+kind: documentation
+description: View anomalies and outliers in context with Watchdog Insights
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-watchdog-insights-log-management/"
+  tag: "Blog"
+  text: "Augmented troubleshooting with Watchdog Insights"
+- link: "https://www.datadoghq.com/blog/watchdog-insights-apm/"
+  tag: "Blog"
+  text: "Automatically detect error and latency patterns with Watchdog Insights for APM"
+---
+
+## Overview
+
+Datadog Watchdog constantly runs in the background, scanning for anomalies in your organization's entire data set. As you navigate the Datadog UI, Watchdog Insights show you the information most relevant to your current context. Using your active product and your active query, Watchdog Insights displays a filtered and sorted list of anomalies.
+
+Investigating an incident requires trial and error. Drawing from their experience, engineers familiar with a particular area know where to first look for potential problems. Using Watchdog Insights allows all engineers, including less experienced ones, to pay attention to the most important data and accelerate their incident investigations.
```

Comment:
The goal of the public documentation is to inform and explain, but not to market or sell Datadog. In this paragraph, I tried to write about the value of Watchdog Insights in lowering the barrier of entry to being effective in investigations.

Samir, can you tell me how Watchdog Insights determines which items are higher priority? That could strengthen our case here about the value of Watchdog Insights without veering into marketing language.

Reply:
Insights are prioritized, from left to right in the Insight's carousel, based on Watchdog evaluating combinations of several factors like: state (ongoing vs resolved), status level (eg for logs: warning vs error vs critical...), the anomaly's start time (latest before earliest), the nature of the anomaly (eg. faulty deployments vs outliers ...)